server/controllers/recommendations/createAndDownloadDocument.ts

```diff
@@ -62,13 +62,17 @@ export const createAndDownloadDocument =
         const isSpoRecordedRationale = statuses.find(status => status.name === STATUSES.SPO_RECORDED_RATIONALE)
         if (!isSpo && isSpoRecordedRationale) {
           activate.push(STATUSES.CLOSED)
+          if (documentType === 'PART_A') {
+            activate.push(STATUSES.SENT_TO_PPCS)
+          }
         }
       }
     }
     if (documentType === 'NO_RECALL_LETTER') {
       const isCompleted = statuses.find(status => status.name === STATUSES.COMPLETED)
       if (!isCompleted) {
         activate.push(STATUSES.COMPLETED)
+        activate.push(STATUSES.DNTR_DOWNLOADED)
```

Comment:
Should this be outside of the if?  Or perhaps even have its own if - if no DNTR_DOWNLOADED status, then add one?

Reply:
I've moved it to it's own if block to follow the existing pattern.